subsys/net/ip/net_shell.c

```diff
@@ -287,6 +287,7 @@ static void print_supported_ethernet_capabilities(
 }
```

Comment:
Looks good. Could you change the commit subject to `net: shell: Wrap iface_flags2str in #ifdef to silence warning` which is more appropriate here.

Reply:
Done :+1: